src/pages/templates/page-sections/pricing/threeTierPricing.tsx

```diff
@@ -0,0 +1,202 @@
+import {
+  Box,
+  Flex,
+  Heading,
+  List,
+  ListIcon,
+  ListItem,
+  Text,
+  VStack,
+  Button,
+  Badge,
+  useColorModeValue,
+} from '@chakra-ui/react';
+import { FaCheckCircle, FaTimesCircle } from 'react-icons/fa';
+
+export default function PricingTable() {
+  const borderBg = useColorModeValue('gray.200', 'gray.500');
+  const pricingItemStyles = {
+    py: 16,
+    px: 8,
+    mb: 4,
+    mt: { base: 12, lg: 24 },
+    mx: { base: 'auto', lg: 4 },
+    borderColor: borderBg,
+    borderRadius: 10,
+    borderWidth: '2px',
+    maxW: '350px',
+  };
+  const priceProps = {
+    fontSize: '5xl',
+    fontWeight: 'bold',
+    my: 3,
+  };
+  const priceTypeProps = {
+    fontSize: '2xl',
+    fontWeight: 'semibold',
+    color: 'gray.500',
+  };
```

Comment:
Instead of defining the styles in an object I would like to create small components for each style. You can see an example in [one of the testimonials](https://github.com/hauptrolle/chakra-templates/blob/main/src/pages/templates/page-sections/testimonials/withSpeechBubbles.tsx)